include/mxnet/c_api.h

```diff
@@ -2005,6 +2005,16 @@ MXNET_DLL int MXKVStoreSendCommmandToServers(KVStoreHandle handle,
                                              int cmd_id,
                                              const char* cmd_body);
 
+
+MXNET_DLL int MXKVStoreSetServerProfilerConfig(KVStoreHandle handle,
```

Comment:
documentation in comments, please

Reply:
Apologies missed this in the cpp files. Added everywhere

Reply:
Apologies. Missed documentation for the cpp parts. Added everywhere